pymeasure/instruments/instrument.py

```diff
@@ -86,6 +87,20 @@ class Instrument(object):
     :param includeSCPI: A boolean, which toggles the inclusion of standard SCPI commands
     """
 
+    # Variable holding the list of DynamicProperty parameters that are configurable
+    # by users
+    special_keys = ('get_command',
+                    'set_command',
+                    'docs',
```

Comment:
You specify `docs` here as being configurable but I wonder if that is actually the case as `docs` is set in line 341 (`fget.__doc__ = docs`), but it seems to me that the docstring cannot be changed afterwards.

Reply:
@CasperSchippers I think you right, my thoughts on this:

- here we are talking about python properties
- python properties are attribute at class level  (and not at instance level)
- attribute documentation can only be accessed at class level
- the method proposed in this PR is not changing the behaviour at class level, but at instance level

As consequence, I would say that is not possible to dynamically change the `docstring` and you are correct.
I would remove `docs` from `special_keys` and accept that docstring` cannot be changed. If this is absolutely needed, it can be addressed by redefining altogether the property (even thought this will force some repetition)